extensions/frontend-tiles/src/GraphicsProvider/GraphicRepresentationProvider.ts

```diff
@@ -0,0 +1,255 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AccessToken, Logger} from "@itwin/core-bentley";
+import { loggerCategory} from "../LoggerCategory";
+
+/** The format of the Graphic Representation
+ * @beta
+ */
+export enum GraphicRepresentationFormat {
+  IMDL = "IMDL",
+  Tiles3D = "3DTILES",
+}
+
+/** Graphic representations are generated from Data Sources.
+ * The status of a Graphic Representation indicates the progress of that generation process.
+ * @beta
+ */
+export enum GraphicRepresentationStatus {
+  InProgress = "In progress",
+  Complete = "Complete",
+  NotStarted = "Not started",
+  Failed = "Failed",
+}
+
+/**
+ * Represents a data source for a graphic representation.
+ * A data source is usually higher-fidelity and contains more information, but may not be as well suited for visualization
+ * as a graphic representation.
+ * @beta
+ */
+export interface DataSource {
+  /** The iTwinId associated with th data source */
+  iTwinId: string;
+  /** The id of the data source */
+  id: string;
+  /** The change id of the data source */
+  changeId?: string;
+  /** The type of the data source */
+  type: string;
+}
+
+/** represents a visual representation of data from a data source.
+ * This could be a 3D model, a map, or any other kind of graphical data.
+ * @see [[queryGraphicsDataSources]].
```

Comment:
An `@see` comment should always explain **why** you should consult the linked documentation (e.g., in this case, because you want to find data available data sources).